scenes/game/game_wrapper.gd

```diff
@@ -1,3 +1,6 @@
+# The wrapper manages signals from a remote player, the ai, or the local player
```

Comment:
it actually doesn't manage anything to do with the AI.
the UI actually hosts the AI, which is probably a terrible design but that's how it is, haha...